crates/bevy_ui/src/layout/debug.rs

```diff
@@ -1,92 +1,17 @@
-use std::fmt::Write;
-
-use taffy::{NodeId, TraversePartialTree};
-
-use bevy_ecs::prelude::Entity;
-use bevy_utils::HashMap;
-
 use crate::layout::ui_surface::UiSurface;
 
-/// Prints a debug representation of the computed layout of the UI layout tree for each window.
+/// Prints a debug representation of the computed layout of the UI layout tree for each camera.
+#[deprecated(
+    since = "0.13.3",
+    note = "use `ui_surface.ui_layout_tree_debug_string()` instead"
+)]
```

Comment:
see: https://github.com/bevyengine/bevy/pull/13359#discussion_r1599145085

If we leave `ui_layout_tree_debug_string` in `UiSurface` then this function should retain the deprecation notice so it can be removed later. 

- [ ] Either Remove the notice or update the version to match whatever version this is slated to land in.